install_config/storage_examples/direct_storage.adoc

```diff
@@ -0,0 +1,481 @@
+spec:
+  containers:
+  - image: fedora/nginx
+    imagePullPolicy: IfNotPresent
+    name: nginx-nfs-pod
+    ports:
+    - containerPort: 80
+      name: web
+      protocol: TCP
+    resources: {}
+    securityContext:
+      privileged: false
+    terminationMessagePath: /dev/termination-log
+    volumeMounts:
+    - mountPath: /usr/share/nginx/html
+      name: nfsvol
+    - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
+      name: default-token-a06zb
+      readOnly: true
+  dnsPolicy: ClusterFirst
+  host: ose70.rh7
+  imagePullSecrets:
+  - name: default-dockercfg-xvdew
+  nodeName: ose70.rh7
+  restartPolicy: Always
+  securityContext:
+    supplementalGroups:
+    - 100003i <3>
+  serviceAccount: default
+  serviceAccountName: default
+  terminationGracePeriodSeconds: 30
+  volumes:
+  - name: nfsvol
+    persistentVolumeClaim:
+      claimName: nfs-claim-prebind <4>
+  - name: default-token-a06zb
+    secret:
+      secretName: default-token-a06zb
+status:
+  conditions:
+  - lastProbeTime: null
+    lastTransitionTime: 2016-03-21T13:59:49Z
+    status: "True"
+    type: Ready
+  containerStatuses:
+  - containerID: docker://a3292104d6c28d9cf49f440b2967a0fc5583540fc3b062db598557b93893bc6f
+    image: fedora/nginx
+    imageID: docker://403d268c640894cbd76d84a1de3995d2549a93af51c8e16e89842e4c3ed6a00a
+    lastState: {}
+    name: nginx-nfs-pod
+    ready: true
+    restartCount: 0
+    state:
+      running:
+        startedAt: 2016-03-21T13:59:49Z
+  hostIP: 192.168.234.148
+  phase: Running
+  podIP: 10.1.0.4
+  startTime: 2016-03-21T13:59:47Z
+
+----
+<1> The SCC used by the pod.
+<2> The project (namespace) name.
+<3> The supplemental group ID for the pod (all containers).
+<4> The PVC name used by the pod.
+====
+
+== [METHOD 2] Go directly to the storage asset using the volume plug-in
+=== 1. Creating the pod definition
+Executing this method, the user may skip the creation of the Persistent Volume (PV) and
+Persistent Volume Claim (PVC) and access the storage via the path defined in the volumes
+section.
+
+
+.Pod Object Definition
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: busybox-nfs-pod <1>
+  labels:
+    name: busybox-nfs-pod   
+spec:
+  containers:
+  - name: busybox-nfs-pod
+    image: busybox <2>       
+    command: ["sleep", "60000"]
+    volumeMounts:
+    - name: nfsvol-2 <3>
+      mountPath: /usr/share/busybox  <4>
+      readOnly: false
+  securityContext:
+    supplementalGroups: [100003] <5>       
+    privileged: false
+  volumes:
+  - name: nfsvol-2   
+    nfs:
+      path: /opt/nfs  <6>   
+      server: nfs.f22 <7>
+
+----
+<1> The name of this pod as displayed by `oc get pod`.
+<2> The image run by this pod.
+<3> The name of the volume. This name must be the same in both the `containers` and `volumes` sections.
+<4> The mount path as seen in the container.
+<5> The group id to be assigned to the container.
+<6> The path to the storage on the NFS server
```

Comment:
some the numbered notes end in a period and others don't. I think the approach in these docs has been to end these sentences with a period.